### PR TITLE
Closes 108 & 120 -  Created ADLB template

### LINKED ISSUE #108: Feature Request:  ADLB template program
### Feature Idea

ADLB template program

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### LINKED ISSUE #120: Bug: Update `_pkgdown.yml` to add ADLB vignette
### What happened?

Update `_pkgdown.yml` to add ADLB vignette

### Session Information

_No response_

### Reproducible Example

_No response_

### PR DESCRIPTION
**Pull request by AndersAskeland**
Closes #120 
Closes #108 
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by AndersAskeland** [comment]
@pharmaverse/admiralmetabolic Any available for a review?

File: inst/templates/ad_adlb.R [hunk 1]
```diff
@@ -0,0 +1,210 @@
+# Name: ADLB
+#
+# Label: Laboratory Analysis dataset for metabolic trials
+#
+# Input: adsl_metabolic, lb_metabolic, admiralmetabolic_advs
```
Comment (starosto) [inline inst/templates/ad_adlb.R:5]:
```suggestion
# Input: lb_metabolic, admiralmetabolic_adsl, admiralmetabolic_advs
```

File: inst/templates/ad_adlb.R [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+adlb <- adlb %>%
+  derive_vars_merged(
+    dataset_add = adsl,
+    new_vars = adsl_vars,
+    by_vars = get_admiral_option("subject_keys")
+  )
+
+# Add parameter information ----
+# Define parameter look-up table used for merging parameter codes to ADLB
+param_lookup <- tribble(
```
Comment (starosto) [inline inst/templates/ad_adlb.R:52]:
```suggestion
param_lookup <- tibble::tribble(
```
or add {tibble} in section # Attach/load required packages

File: inst/templates/ad_adlb.R [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  "GGT", "GGT", "Gamma Glutamyl Transferase (U/L)", 5,
+  "GLUC", "GLUC", "Glucose (mmol/L)", 6,
+  "HBA1CHGB", "HBA1CHGB", "Hemoglobin A1C/Hemoglobin (mmol/mol)", 7,
+  "INSULIN", "INSULIN", "Insulin (mIU/L)", 8,
+  "TRIG", "TRIG", "Triglycerides (mg/dL)", 9
+)
+
+# Add parameter (PARAMCD) info to enable later ADLB derivations. Additional
+# parameter information will be merged again, after all AVDS derivations are
+# completed.
```
Comment (starosto) [inline inst/templates/ad_adlb.R:67]:
```suggestion
# Add parameter (PARAMCD, PARAM, PARAMN) info to enable later ADLB derivations.
```

**Review by starosto — CHANGES_REQUESTED** [review]
Hi @AndersAskeland, it looks really good. I left some minor commnets. Please consider to add ABLFL, BASE, CHG, PCHG, ASEQ.

Also, typical for ADLB varables are ANRIND, R2BASE, R2ANRLO, R2ANRHI, SHIFT, flag variables (ANL01FL, LVOTFL), 
minimum value, maximum value - but the code to derive all of them is already in https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adlb.R

**Comment by AndersAskeland** [comment]
> Hi @AndersAskeland, it looks really good. I left some minor commnets. Please consider to add ABLFL, BASE, CHG, PCHG, ASEQ.
> 
> Also, typical for ADLB varables are ANRIND, R2BASE, R2ANRLO, R2ANRHI, SHIFT, flag variables (ANL01FL, LVOTFL), minimum value, maximum value - but the code to derive all of them is already in https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adlb.R

Thanks for the review. Yeah, I actually cant decide if we want the entire template or keep the focus on what is "custom" for `admiralmetabolic`. 

Perhaps we can keep it as is now, and then add an issue, so that we can consider it later. What do you think @starosto ?

File: inst/templates/ad_adlb.R (hunk 2 above)
Comment (AndersAskeland) [reply inst/templates/ad_adlb.R:52]: Ill add to top :)

**Review by starosto — APPROVED** [review]
